packages/ui/src/layout/wui-overlay/styles.ts

```diff
@@ -0,0 +1,14 @@
+import { css } from 'lit'
+
+export default css`
+  :host {
+    top: 0;
```

Comment:
I'd suggest to use `inset: 0;` instead of `top`, `right`, `bottom`, `left` separately. Saves a few lines of code.